langserver/signature.go

```diff
@@ -37,6 +37,28 @@ func (h *LangHandler) handleTextDocumentSignatureHelp(ctx context.Context, conn
 	if activeParameter > 0 {
 		activeParameter = activeParameter - 1
 	}
+
+	filename := h.FilePath(params.TextDocument.URI)
```

Comment:
All this is already done in `typecheck()`. 
We can just return the `start` value from `typecheck()`
But before I went about doing that I wanted to know if there is any simpler way of getting the `Pos` equivalent of given `lsp.Position` ?